client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -3,21 +3,20 @@
 /**
  * Internal dependencies
  */
-
 import { bypassDataLayer } from './utils';
-import { mergeHandlers } from 'state/action-watchers/utils';
+import { getHandlers, registerHandlers } from 'state/data-layer/handler-loading';
 import wpcomHttpHandlers from './wpcom-http';
 import httpData from './http-data';
 import httpHandlers from 'state/http';
 import thirdPartyHandlers from './third-party';
 import wpcomHandlers from './wpcom';
 
-const mergedHandlers = mergeHandlers(
-	httpData,
-	httpHandlers,
-	wpcomHttpHandlers,
-	thirdPartyHandlers,
-	wpcomHandlers
+registerHandlers(
```

Comment:
this call doesn't match. `registerHandlers` expects `( id, handlers )`

Reply:
derp. fixed.